src/styles/_modals.scss

```diff
@@ -66,8 +66,8 @@
   top: calc(max(50%, var(--height) / 2) - var(--height) / 2 + 2em);
 
   &.help-modal {
-    --height: 27em;
-    --width: 57em;
+    --height: 26em;
+    --width: 55.5em;
```

Comment:
it was that wide so that none of the keys in the bottom row became narrower than usual in the widest possible screen?

height is too short for me; it's giving a scroll bar xp

Reply:
I think we should find a better way to represent these. I know one way that can work is to put relative widths as properties (say a normal key is 1 but a shift key is 2.5), and then do true pixel-based calculations. This has the added effect that relative sizes are scale-invariant.

Reply:
oh yeah. asdfgasdkf that sthe morally correct way

somewhat related thought: so that we dont have to worry about smaller screens, have a button to expand the half keyboard to full keyboard, and default to half/full depending on whether you're on mobile or not??? actually debatable how useful this is

Reply:
i don't think this is a good idea. i have to fight with my devtools to get the screen so small that it's ugly and i cannot at all get it to be unusable (except for the ` key on extremely small screens). like i really don't think keys being small is ever an issue, even on mobile in *portrait* mode.

if you need data for this we can read the google accessibility guidelines for minimum button size and compare it to the actual rendered size on a bunch of devices. with the potential exception of the ` key, i'm sure keys will be significantly larger than necessary.

Reply:
also, i'd expect that the people who care about bindings on mobile are highly correlated with those who want as many possible bindings as they can get

Reply:
iirc toolbar bindings are already below minimum button size so :/ :/ :/

this is fair

Reply:
anyway, i think we should just expand this now so that we can merge and open a new issue? xp can you handle that